public/docs/_examples/toh-6/ts/app/heroes.component.css

```diff
@@ -57,3 +58,11 @@ button {
 button:hover {
   background-color: #cfd8dc;
 }
+/* #docregion delete-hero */
+button.delete-button{
+  float:right;
+  background-color: gray !important;
+  color:white;
+}
+/* #enddocregion delete-hero */
+/* #enddocregion */
```

Comment:
Drop the last two lines (L67-68) since all regions are automatically closed at the end of the file (and omitting them makes for fewer of those extra `docregion` tags! :).